specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2024-03-01-preview/purviewdatamap.json

```diff
@@ -6866,11 +6941,140 @@
+        "assetType",
+        "azureResourceHierarchy"
+      ],
+      "x-ms-enum": {
+        "name": "NavigationMode",
+        "modelAsString": true
+      }
+    },
+    "ItemPath": {
+      "type": "object",
+      "properties": {
+        "path": {
+          "type": "string",
+          "description": "The identifier of the item. The path starts with an empty string or a single slash. To navigate further, you need to concatenate with a slash to form the full itemPath using the previous navigation response's relative item path."
+        },
+        "extendedProperties": {
+          "type": "object",
+          "description": "The extended properties of the itemPath are typically obtained from the last navigation response. While not mandatory to provide, including them can enhance performance. Otherwise, there may be some impact on performance.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      },
+      "description": "The identifier of navigation request."
+    },
+    "NavigationResult": {
+      "type": "object",
+      "properties": {
+        "items": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NavigationElement"
+          }
+        },
+        "continuationToken": {
+          "type": "string",
+          "description": "The token used to get next batch of data. Absent if there's no more data."
+        }
+      },
+      "description": "The response payload of the Navigation API."
+    },
+    "NavigationElement": {
+      "type": "object",
+      "description": "The item payload of the NavigationResult.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the item."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type name of the item. Eg. EntityType."
+        },
+        "isLeafNode": {
+          "type": "boolean",
+          "description": "Indicates whether the item is a leaf node. If it is, further navigation is not possible."
+        },
+        "isEntity": {
+          "type": "boolean",
+          "description": "Whether the item is Entity. If yes, can get complete definition of an entity given its itemPath"
+        },
+        "itemPath": {
+          "$ref": "#/definitions/ItemPath"
+        },
+        "count": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The count of the top level asset. Won't return if request payload 'includeNextLevelAssetCount' is false."
+        },
+        "properties": {
+          "type": "object",
+          "description": "The additional properties of the navigation element.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
     "LastModifiedTS": {
       "type": "string",
       "title": "lastModifiedTS",
       "description": "ETag for concurrency control."
     },
+    "Error": {
```

Comment:
Nit: we typically call this `ErrorResponse`. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#rest-error-response-body-structure

Reply:
The name `ErrorResponse` has already been taken by previous code. Its response payload does not fully comply with the Azure REST API requirements, due to the influence of the native API from the Apache Atlas open-source product. Do I need to deliberately change the naming of `ErrorResponse` in other APIs and adapt this `ErrorResponse` for my use?

Reply:
updated to `ErrorResponse`